src/Components/FundingOrgs/FundingOrgsTable.js

```diff
@@ -0,0 +1,90 @@
+import React, { useMemo, useState } from "react";
+import { useTable } from "react-table";
+import "./FundingOrgsTable.css";
+import formatDate from "../../Helpers/formatDate";
+
+//fontawesome
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faTrashAlt } from "@fortawesome/free-solid-svg-icons";
+import { faEdit } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useCurrentOrganizationContext } from "../../Contexts/currentOrganizationContext";
+
+library.add(faTrashAlt);
+library.add(faEdit);
+
+export default function FundingOrgsTable(props) {
+  const [showFundingOrgEdit, setShowFundingOrgEdit] = useState(false);
+  const handleClose = () => {
+    setShowFundingOrgEdit(false);
+  };
+
+  const columns = useMemo(
+    () => [
+      { Header: "Name", accessor: "name" },
+      { Header: "Website", accessor: "website" },
+      {
+        Header: "Date Created",
+        accessor: (row) => formatDate(row.created_at),
+      },
+      {
+        Header: "Last Modified",
+        accessor: (row) => formatDate(row.updated_at),
+      },
```

Comment:
Wanted to confirm that these are the columns we want. The wireframe there are "Funding Org Name" and "Funding Org Name"

Reply:
Good catch - I think we can do name and website since they're indexed on the header. I'll make a note to confirm with Eli.